src/app/main-overview/components/ControlPlaneDetails.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div class="stack">
+    <KCard>
+      <template #body>
+        <div class="card-header">
+          <div class="card-title">
+            <h2>{{ t('main-overview.detail.health.title') }}</h2>
+          </div>
+        </div>
+
+        <div class="variable-columns">
+          <ResourceStatus
+            :total="store.getters['config/getMulticlusterStatus'] ? props.zoneOverviews.length : 1"
```

Comment:
note: it's the one that we saw https://github.com/kumahq/kuma-gui/pull/1291/files#diff-61c9d2287811884ae04444895cae863a245228372fdf8ccfea08a2b9def96e62R47